website/config.toml

```diff
@@ -127,3 +127,16 @@ name = "GitHub"
 weight = 99
 url = "https://github.com/awslabs/karpenter"
 pre = "<i class='fab fa-github'></i>"
+
+# Add your release versions here
+[[params.versions]]
+  version = "latest"
+  url = "https://karpenter-versions.gcline.us/docs/"
```

Comment:
Assuming this will get cleaned up before submitting?

Reply:
yup! it's just so my preview https://karpenter-versions.gcline.us/docs/ works 

https://github.com/awslabs/karpenter/pull/801#issuecomment-973763810